db/migrate/20161007174137_create_urls_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUrlsTable < ActiveRecord::Migration
+  def change
+    create_table :urls do |t|
+
+      t.string :url
```

Comment:
be sure to name the attributes in a table differently than the table itself. this can get very confusing very quickly. it's also harder to read